deploy_helm.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# Added color constants
+RED='\033[0;31m'
+NC='\033[0m'
+
+# Add announcement that this script has just started with timestamp
+echo -e "${RED}Script started at $(date +%T)${NC}"
+
+while getopts ":e:n:" o; do
+    case "${o}" in
+        e) env_name=$OPTARG;;
+        n) chart_name=$OPTARG;;
+        *) echo "No reasonable options found"
+    esac
+done
+
+# Create storage-class if not exists
+get_sc=$(kubectl get sc "happy-bank-local-storage" --no-headers 2>/dev/null)
+if [ -n "${get_sc}" ]; then
+    echo -e "${RED}Storage class exists:${NC}"
+    kubectl get sc
+else
+    echo "Creating storage class"
+    kubectl apply -f kubernetes/storage-class.yml
+fi
+
+# Create namespace
+get_ns=$(kubectl get ns "$env_name" --no-headers 2>/dev/null)
```

Comment:
If env_name var doesn't exists, probably it will crash. We should add reading context from kube config and get its namespace.